pkgs/development/python-modules/scikitlearn/default.nix

```diff
@@ -54,16 +54,30 @@ buildPythonPackage rec {
     joblib
     threadpoolctl
   ];
-  checkInputs = [ pytest ];
+  checkInputs = [ pytestCheckHook ];
+
+  preCheck = ''
+    export OMP_NUM_THREADS=1
+    export HOME=$(mktemp -d)
+    cd "$HOME"
+  '';
+
+  pytestFlagsArray = [
+    "--pyargs sklearn"
+    # verbose build outputs needed to debug hard-to-reproduce hydra failures
+    "-v"
+  ];
+
+  disabledTests = [
+    # test causes segmentation fault for hydra builds Probably related to a
+    # long-standing joblib issue (https://github.com/joblib/joblib/issues/563).
+    # See also: https://github.com/scikit-learn/scikit-learn/issues/17582
+    "test_estimators[NuSVC()-check_estimators_fit_returns_self(readonly_memmap=True)]"
+  ];
 
   LC_ALL="en_US.UTF-8";
 
   doCheck = !stdenv.isAarch64;
```

Comment:
Do we know if this is still necessary? If you remove it, I can have r-rmcgibbo give it a try on aarch64.

Reply:
I don't know. It's a bit out of the scope of the original intent of the PR which was to fix the darwin builds. My fear is that it breaks, and then I have to change the code again and re-push, etc. Could follow-up on this at a later point?

Reply:
Yes, that's fine with me.